server/types/base_ids.go

```diff
@@ -129,6 +141,16 @@ var preferredTypeInCategory = map[TypeCategory]DoltgresTypeBaseID{
 	TypeCategory_StringTypes:  Text.BaseID(),
 }
 
+// TypeAlignment represents the alignment required when storing a value of this type.
+type TypeAlignment string
+
+const (
+	TypeAlignment_Char   TypeAlignment = "c"
+	TypeAlignment_Short  TypeAlignment = "s"
+	TypeAlignment_Int    TypeAlignment = "i"
+	TypeAlignment_Double TypeAlignment = "d"
+)
```

Comment:
Where are these used?

Reply:
Oh I see now. Might as well also add these to the `DoltgresType` interface too, rather than returning hard-coded ones in the row iter.